src/Form.js

```diff
@@ -0,0 +1,38 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import bootstrapUtils, { bsClass } from './utils/bootstrapUtils';
+
+const propTypes = {
+  horizontal: React.PropTypes.bool,
+  inline: React.PropTypes.bool,
+};
+
+const defaultProps = {
+  horizontal: false,
+  inline: false,
+};
+
+class Form extends React.Component {
+  render() {
+    const { horizontal, inline, className, ...props } = this.props;
+    delete props.bsClass;
+
+    const classes = [];
+    if (horizontal) {
+      classes.push(bootstrapUtils.prefix(this.props, 'horizontal'));
+    }
+    if (inline) {
+      classes.push(bootstrapUtils.prefix(this.props, 'inline'));
+    }
+
+    return (
+      <form {...props} className={classNames(className, classes)} />
```

Comment:
I mean besides the obvious semantic reason, here isn't any need for this to be an actual `<form>` component, perhaps support `componentClass`? I find I mix and match inline and normal "forms" often enough that I wouldn't mind it, but maybe that's too ugly?